### PR TITLE
pv 1.6.19

### PR DESCRIPTION
**Pull request by moonfruit**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by cho-m** [comment]
Probably missing include of header that defines `NULL` like `stddef.h`, `stdio.h`, etc:
```
src/pv/number.c:32:6: error: use of undeclared identifier 'NULL'
        if (NULL == str)
            ^
src/pv/number.c:122:6: error: use of undeclared identifier 'NULL'
        if (NULL == str)
            ^
```

File: Formula/pv.rb [hunk 1]
```diff
@@ -53,3 +53,15 @@ def install
  #ifdef __cplusplus
  extern "C" {
  #endif
+diff --git a/src/pv/number.c b/src/pv/number.c
```
Comment (carlocab) [inline Formula/pv.rb:56]: Has this new patch been upstreamed?

Comment (moonfruit) [reply Formula/pv.rb:56]: No, that is added by mine, and not report to upstream yet.

Comment (moonfruit) [reply Formula/pv.rb:56]: OK, now I report this to `pv` via [Contact](http://www.ivarch.com/personal/contact.shtml). What should I do next?

**Review by carlocab — APPROVED** [review]
Thanks @moonfruit.

**Pull request closed without merging**